site/content/en/docs/administration/basics/installation.md

```diff
@@ -715,6 +721,12 @@ export CVAT_HOST=<YOUR_DOMAIN>
 export ACME_EMAIL=<YOUR_EMAIL>
 ```
 
+Optional health check configuration:
+
+```shell
+export CVAT_HEALTH_DISK_USAGE_MAX=90
+```
```

Comment:
If this variable is not handled in `docker-compose.yml`, this will only work in a bare-metal deployment. As I wrote in https://github.com/cvat-ai/cvat/pull/9870#discussion_r2410913025, consider either adding it there or mention the required adjustments.